src/Sylius/Bundle/AdminBundle/composer.json

```diff
@@ -70,9 +69,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Sylius\\Bundle\\AdminBundle\\": "",
-            "Sylius\\Bundle\\AdminBundle\\Tests\\": "Tests/",
-            "Sylius\\Bundle\\AdminBundle\\Application\\": "test/src/"
+            "Tests\\Sylius\\Bundle\\AdminBundle\\": "tests/"
```

Comment:
You've created tests in the `Tests` directort and have not changed the namespaces for the existing ones, so you can leave `Tests/` here for now